scripts/update-projects-version.py

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/env python3
```

Comment:
@mpryc please test out this script, once validated I will delete `scripts/create_release_pr.sh`, `scripts/specify_operator_update.py` and `scripts/create_pelorus_operator.sh` and use this one to do their job